pkgs/tools/misc/grex/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, fetchpatch, rustPlatform, cmake, perl, pkgconfig, zlib
+, darwin, libiconv, installShellFiles
+}:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  pname = "grex";
+  version = "1.1.0";
+
+  cargoSha256 = "0kf2n2j7kfrfzid1h2gd0qf53fah0hpyrrlh2k5vrhd0panv3bwc";
+
+  src = fetchFromGitHub {
+    owner = "pemistahl";
+    repo = "grex";
+    rev = "v${version}";
+    sha256 = "1viph7ki6f2akc5mpbgycacndmxnv088ybfji2bfdbi5jnpyavvs";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig perl installShellFiles ];
+  buildInputs = [ zlib ]
+  ++ stdenv.lib.optionals stdenv.isDarwin [
+    libiconv darwin.apple_sdk.frameworks.Security ]
+  ;
```

Only Security seems to be needed.

```suggestion
  buildInputs =  stdenv.lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
```